force-app/main/educationCloudSettings/lwc/edcReleaseManagementCard/edcReleaseManagementCard.html

```diff
@@ -22,14 +22,36 @@ <h3 slot="title" id="cardTitle">
                 <p>{labelReference.releaseManagementDescription}</p>
             </div>
             <div class="slds-col slds-small-size_1-of-1 slds-medium-size_6-of-12 slds-large-size_4-of-12">
-                <lightning-button
-                    label={labelReference.releaseManagementButton}
-                    title={labelReference.releaseManagementButtonA11y}
-                    aria-described-by="cardTitle"
-                    class="btnGoToReleaseManagement slds-float_right"
-                    variant="brand"
-                    onclick={handleNavigationClick}
-                ></lightning-button>
+                <div class="slds-float_right">
+                    <div class="slds-grid slds-grid_vertical">
+                        <div class="slds-col slds-size_1-of-1">
+                            <div if:false={allReleaseGatesActive} class="notification-bell">&#x1F514;</div>
+                            <lightning-button
+                                label={labelReference.releaseManagementButton}
+                                title={labelReference.releaseManagementButtonA11y}
+                                aria-described-by="cardTitle"
+                                class="btnGoToReleaseManagement slds-float_left"
```

Comment:
Consider adding a QA Locator data attribute to this item. Using the class to find it is something that we would use internally, but for browser testing, the data attribute is our tool of choice.

Reply:
Done, added "data-qa" attributes in a few palces instead